Omnibus.md

```diff
@@ -591,84 +592,97 @@ attack the unsuspecting looters. If you are unsure as to whether someone's
-number of weapons used. Some monsters and characters under certain spells may
-require more scalping blows to completely destroy. If you strike 200 blows and
-the victim says "The job is not yet done," then the scalping is not yet
-complete. Some spells, such as [Strange Brew: Potion of Acid](#potion-of-acid)
-and [Assassin's Blade](#assassins-blade), may accelerate the act of scalping.
-
-After the scalping is finished, the player of the scalped character must present
-their character's scalp to the character that just scalped them. It is then that
-player's responsibility to present the scalp to an EH or an appointed marshal.
-The EH must be informed immediately of the scalping, thus allowing the EH time
-to prepare for those who might wish to return the scalped character to life.
-
-Characters that are pretending to be dead should interpret any scalping blows as
-blows to the closest kill location. So, if your PC is pretending to be dead and
-someone starts to scalp them, the first blows that are struck on the ground next
-to you should be played as if they were striking your PC on the nearest kill
-location, destroying any undamaged armor, and then killing them.
-
-If your PC is killed and scalped, they will not remember anything about how they
-died should they manage to be raised.
-
-Should your PC be dead at the end of an event without being raised, even if the
-character was not actively scalped, the PC will be considered scalped.
-
-### Fixing a Scalped Character ### {#fixing-a-scalped-character}
-
-To restore someone to life after scalping, characters must first have the body
-of the person needing to be raised. Then characters must either cast a [Call the
-Soul](#call-the-soul), an [Intervention](#intervention) spell, or administer a
-[Potion of Soul Snare](#potion-of-soul-snare) (see [Strange
-Brew](#strange-brew)) to summon and reattach the scalp. If characters lack the
-body, only an [Intervention](#intervention) spell will be able to return the
-scalped PC back to life. When the body and scalp are reunited, characters must
-cast one [Raise Dead](#raise-dead) spell for each event (including the first)
-since the scalped PC was scalped in order to raise them. A different spellcaster
-must provide each [Raise Dead](#raise-dead) spell used for this purpose.
+## 3.5: [Soul Loss]{.new} ## {#soul-loss}
+
+[Soul loss]{.new} is the removal of a character's life essence. It is a
+more serious form of death that requires more than a simple spell to repair.
+[A character who has lost their soul]{.new} cannot be raised or animated
+until their soul has been restored.
+
+### [Soul Token]{.new} ### {#soul-tokens}
+
+A [soul token]{.new} is a token carried by each player, which represents
+the life essence of that player's character. Players must be carrying their PC's
+[soul token]{.new} at all times, unless the PC is dead and
+[soulless]{.new} (see below), or unless they are under the influence of a
```

Comment:
Other (see below) instances have been linked. Is that unnecessary since the section is immediately below? @3geek14

Reply:
Additionally:
`[soulless]{.new} (see below) or unless they are under the influence of a`